esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -0,0 +1,571 @@
+# ESMValTool
```

Comment:
This is a cool idea and should be very helpful when adding new observational data sets!